src/ui/win32/bindings/MemoryViewerControlBinding.cpp

```diff
@@ -264,6 +268,15 @@ void MemoryViewerControlBinding::OnClick(POINT point)
     Invalidate();
 }
 
+void MemoryViewerControlBinding::OnShiftClick(POINT point)
+{
+    OnClick(point);
```

Comment:
Note that `OnClick` will abort if the memory viewer has a fixed address (such as when it's used in the pointer finder). This functionality should also be non-functional in the pointer finder.